app/api/files/route.ts

```diff
@@ -0,0 +1,21 @@
+import { put } from '@vercel/blob'
+import { NextResponse } from 'next/server'
+import { customAlphabet } from 'nanoid'
+
+export const runtime = 'edge'
+
+const nanoid = customAlphabet(
+  '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz',
+  7,
+) // 7-character random string
+export async function POST(req: Request) {
+  const file = req.body || ''
+  const contentType = req.headers.get('content-type') || 'text/plain'
```

Comment:
**nitpick:** We should validate file types so nothing else is being uploaded. Let's create some collection of available content types. For now we should be able to upload only images